ntoskrnl/io/iomgr/irq.c

```diff
@@ -185,7 +187,7 @@ IopConnectInterruptExFullySpecific(
                                 Parameters->FullySpecified.ShareVector,
                                 Parameters->FullySpecified.ProcessorEnableMask,
                                 Parameters->FullySpecified.FloatingSave);
-    DPRINT("IopConnectInterruptEx_FullySpecific: has failed with status %X", Status);
+    DPRINT1("IopConnectInterruptExFullySpecific: has failed with status 0x%lx", Status);
```

Comment:
I am notorously bad with format strings. Therefore I do have to ask: What do you want to improve with the changed format-string? Was the old one really wrong? And is that a realistic thing to ever fear in practice (any overflow can happen) or is that just knee-jerking?

Reply:
Consistency only: `0x%lx` (or `0x%08lx`) is usual for `NTSTATUS`.

Reply:
Consistency only is not a good argument to go around changing stuff like this. But MSVC /analyze mode will in fact warn about this, so it's good to quiet it down.